src/main/java/edu/cornell/kfs/module/receiptProcessing/CsvBatchInputFileTypeBase.java

```diff
@@ -113,6 +115,15 @@ public Object parse(byte[] fileByteContent) throws ParseException {
         return convertParsedObjectToVO(parsedContents);
     }
 
+    protected void validateCSVHeader(byte[] fileByteContent, List<String> headerList) throws ParseException {
+        try (ByteArrayInputStream validateFileContents = new ByteArrayInputStream(fileByteContent)) {
+            validateCSVFileInput(headerList, validateFileContents);
```

Comment:
There are some potential resource leaks in the validateCSVFileInput() method that also need to be fixed. That method currently creates its own InputStreamReader and CSVReader but never closes them.

(Note that, due to the setup of the data that this class is being used for, the validation can succeed even though its reader doesn't have the same CSV parser setup as the main data-reading code.)